ui/apps/platform/src/Containers/Collections/CollectionForm.css

```diff
@@ -16,4 +16,13 @@
 .collection-form-expandable-section button.pf-c-expandable-section__toggle:focus {
   /* This blue outline on focus is a global problem, but it is especially jarring in this case */
   outline: none;
+}
+
+.collection-form-expandable-section .pf-c-expandable-section__toggle-icon {
+  /*
+    Some conflict with TW styles in production build only is causing this element to be
+    misaligned due to an excessive calculated height. Setting this max prevents it from
+    growing too large. This likely can be removed when we fully migrate to PF.
+  */
+  max-height: 2rem;
```

Comment:
Can you add empty line at the end of the file?

Reply:
I'll piggyback this on my next PR to avoid an additional CI run.